tms/__main__.py

```diff
@@ -64,53 +28,19 @@ async def watcher_loop() -> None:
         for jel_fpath in ENV.JOB_EVENT_LOG_DIR.iterdir():
             if jel_fpath in in_progress:
                 continue
-            task = asyncio.create_task(watcher.watch_job_event_log(jel_fpath, ewms_rc))
+            task = asyncio.create_task(
+                watcher.watch_job_event_log(
+                    jel_fpath,
+                    ewms_rc,
+                    tmonitors,
+                )
+            )
             await asyncio.sleep(0)  # start above task
```

Comment:
I wonder if this is actually necessary?  They'll all get started when we hit the sleep below.

Reply:
With the sleep-0 here, the task starts up before the `iterdir()` yields another. That's probably not needed since that wait is negligible, but that was my intention. I'll remove it